daphne/src/roles.rs

```diff
@@ -61,9 +61,15 @@ pub trait DapAggregator<S>: HpkeDecrypter + Sized {
     /// Decide whether the given DAP request is authorized.
     async fn authorized(&self, req: &DapRequest<S>) -> Result<bool, DapError>;
 
+    /// Look up the DAP global configuration.
+    fn get_global_config(&self) -> &DapGlobalConfig;
+
     /// Look up the DAP task configuration for the given task ID.
     fn get_task_config_for(&self, task_id: &Id) -> Option<&DapTaskConfig>;
 
+    /// Get the current time.
```

Comment:
We should document the format of the time (number of seconds since UNIX epoch).

Reply:
Sounds good. I've updated the comment.